inception-search-core/src/main/java/de/tudarmstadt/ukp/inception/search/index/PhysicalIndex.java

```diff
@@ -28,21 +28,21 @@
 import de.tudarmstadt.ukp.inception.search.ExecutionException;
 import de.tudarmstadt.ukp.inception.search.SearchResult;
 
-public interface Index
+public interface PhysicalIndex
```

Comment:
What is a physical index?

Reply:
Indexes have two different levels of abstraction, the logical and the physical. This is the physical level of abstraction.